dsgrid/dataset/dataset_schema_handler_standard.py

```diff
@@ -70,7 +71,7 @@ def make_project_dataframe(self):
         )
         # TODO: handle fraction application
         # Currently this requires fraction = 1.0
-        ld_df = ld_df.join(lk_df, on="id").drop("id")
+        ld_df = ld_df.join(lk_df, on="id")
```

Comment:
Keeping `id` will be problematic if/when we take a union of datasets where one is in single-table format. Also, it could be confusing to users about when a dataset has `id` and when it does not.

Reply:
we should always keep id

Reply:
yeah I see your point. "id" are so useful though. We should require it in single-table as well.

Reply:
I don't like the idea of "id" in the one-table format. It seems highly arbitrary and easy to overwrite.